xDrip/Model/Device/CGMDevice.swift

```diff
@@ -144,15 +144,21 @@ final class CGMDevice: Object {
     }
     
     var transmitterStartDate: Date? {
-        get {
-            let transmitterData = CGMDevice.current.metadata(ofType: .transmitterTime)
-            guard let ageString = transmitterData?.value,
-                let age = Double(ageString),
-                let date = transmitterData?.date else {
-                    return nil
-            }
-            return date - age
+        let transmitterData = CGMDevice.current.metadata(ofType: .transmitterTime)
+        guard let ageString = transmitterData?.value,
+              let age = Double(ageString),
+              let date = transmitterData?.date else {
+            return nil
+        }
+        return date - age
+    }
+    
+    var isFirstTransmitterVersion: Bool? {
```

Comment:
I don't like such approach:
1. This logic should only apply for Dexcom G6. What if we add other transmitters in the future?
2. What if Dexcom will come up with a third version? I would make it an int and return an integer value of the first digit.